regolith/mongoclient.py

```diff
@@ -5,6 +5,10 @@
 import shutil
 import subprocess
 from glob import iglob
+from warnings import warn
+
+warn("Mongo support will be deprecated in the near future please use "
+     "fsclient.FileSystemClient")
```

Comment:
I think the warning should only be presented if the client is instantiated, not on import...